src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -580,10 +577,10 @@ private void onPinnedEndpointChangedEvent(
         // Find the new pinned endpoint.
         String newPinnedEndpointID = (String) jsonObject.get("pinnedEndpoint");
 
-        List<String> newPinnedIDList = Collections.EMPTY_LIST;
+        Set<String> newPinnedIDList = Collections.EMPTY_SET;
```

Comment:
I should've changed the variable names, thanks @bbaldino. It is true that we only parse a single ID; meet is sending a single ID at a time, but the JVB code is ready to handle the general case where you have a single data channel message with multiple pinned/selected endpoints.